### DISCUSSION TITLE
My Friend is a GHOST in github!

### DISCUSSION THREAD

**Discussion by juniorrondini** [community/community#76901] [Other Feature Feedback, Questions, & Ideas] [answered]
## GHOST FRIEND

Bug

Guy's, when my friend make a commit he desapear have name but don't have link for perfil or picture in profile... how resolve this

![imagem_2023-11-24_174920495](https://github.com/community/community/assets/65919281/dae75f32-3239-4b5d-9307-69453f3e5a32)

**Comment by milad176** [comment]
If your friend's commits are not showing their name, profile picture, or link to their profile on GitHub, here are simplified steps to troubleshoot:

1. Git Configuration:
   Make sure your friend has configured Git with the correct name and email matching their GitHub account:

   git config --global user.name "Your Name"
   git config --global user.email "your.email@example.com"
   

2. GitHub Email Verification:
   Verify that the email associated with Git commits is added and verified in their GitHub account settings.

3. Commit Authorship:
   Ensure commits are made with the email linked to their GitHub account.

4. Check Commit History:
   Review commit history on GitHub for any unverified or different email addresses.

5. Clear Git Cache:
   Clear the local Git credentials cache:

   git credential-cache exit
   
   Make a new commit and push.

6. Profile Privacy Settings:
   Check GitHub profile privacy settings to ensure the email address is visible.

If issues persist, your friend can contact GitHub Support for further assistance. Always refer to the latest GitHub documentation for accurate information.

**Comment by juniorrondini** [comment, marked as answer]
I can't mark your answer as correct, as the option isn't appearing, it worked perfectly, thank you! @milad176
